### PR TITLE
[WIP] RHDEVDOCS 5172 Adding sections from main to GitOps-docs 1.8 branch and shallow restructure

### PR DESCRIPTION
**Pull request by Srivaralakshmi**
**Version(s):** gitops-docs, gitops-docs-1.8, gitops-docs-1.9

**Issue:**
[RHDEVDOCS-5172](https://issues.redhat.com//browse/RHDEVDOCS-5172)

**Link to docs preview:** https://64804--docspreview.netlify.app/openshift-gitops/latest/understanding_openshift_gitops/what-is-gitops.html

**Peer review:** @Preeticp, @mramendi 
**PM review**: @harrietgrace 

**Additional Info:** While it is a size XXL there is no significant new content. Being a shallow restructure, most of the size is due to the creation of multiple directories and moving assemblies and modules into them.  These files are seen in the PR as new but the content is all from the pre-existing assemblies/modules. Only necessary edits were made to the metadata and anchor IDs to rectify the legacy content errors to keep them in line with the OCP docs, mod docs, and RH SSG guidelines. 

This is a restructure that is part of enabling the standalone documentation set for OpenShift GitOps. It was approved by the PM and Content Strategist. It does not affect the main at all, only gitops-docs, gitops-docs-1.8, and gitops-docs-1.9. The PR does not change documentation content, except copying modules or assemblies from the main that were already reviewed for merging to the main.

### CODE REVIEW

**Comment by Srivaralakshmi** [comment]
@Preeticp and @mramendi: could you please do a peer review ack on this PR? Thank you!

**Comment by mramendi** [comment]
Working preview link: https://64804--docspreview.netlify.app/openshift-gitops/latest/understanding_openshift_gitops/what-is-gitops.html

(You will need a separate PR to main to make this the default)

File: modules/gitops-configuring-workloads.adoc [hunk 1]
```diff
@@ -1,12 +1,11 @@
 // Module is included in the following assemblies:
 //
-// * cicd/gitops/configuring-resource-quota.adoc
+// * managing_resource/configuring-resource-quota.adoc
 
 :_content-type: PROCEDURE
 [id="configuring-workloads_{context}"]
 = Configuring workloads with resource requests and limits
 
-[role="_abstract"]
```
Comment (mramendi) [inline modules/gitops-configuring-workloads.adoc:9]: Why this removal?

File: modules/logging-in-to-the-argo-cd-instance-by-using-the-argo-cd-admin-account.adoc [hunk 2]
```diff
@@ -17,21 +16,21 @@
 
 . In the *Administrator* perspective of the web console, navigate to *Operators* -> *Installed Operators* to verify that the {gitops-title} Operator is installed.
 . Navigate to the {rh-app-icon} menu -> *OpenShift GitOps* -> *Cluster Argo CD*. The login page of the Argo CD UI is displayed in a new window.
-. Obtain the password for the Argo CD instance:
+. Optional: To log in with your {OCP} credentials, ensure you are a user of the `cluster-admins` group and then select the `LOG IN VIA OPENSHIFT` option in the Argo CD user interface.
++
+[NOTE]
+====
+To be a user of the `cluster-admins` group, use the `oc adm groups new cluster-admins <user>` command, where `<user>` is the default cluster role that you can bind to users and groups cluster-wide or locally. 
+====
```
Comment (mramendi) [inline modules/logging-in-to-the-argo-cd-instance-by-using-the-argo-cd-admin-account.adoc:24]: This looks like new content, did you take it from another module? Any actual new technical content, even as small as this one, needs SME/QE review.

**Comment by mramendi** [comment]
I've put in some minor notes.

File: modules/gitops-configuring-workloads.adoc (hunk 1 above)
Comment (Srivaralakshmi) [reply modules/gitops-configuring-workloads.adoc:9]: This is not required actually. This was part of Jupiter work efforts, but I received in the OCP team review comments later on that we can remove it. Hence removing it now.

File: modules/logging-in-to-the-argo-cd-instance-by-using-the-argo-cd-admin-account.adoc (hunk 2 above)
Comment (Srivaralakshmi) [reply modules/logging-in-to-the-argo-cd-instance-by-using-the-argo-cd-admin-account.adoc:24]: It is part of the bug that is already merged and closed. Just merging it from the main.

**Review by mramendi — APPROVED** [review]

**Comment by Preeticp** [comment]
The top level restructure looks good to me. A suggestion, if you have the bandwidth, can we club the logging and monitoring sections under Observability like how we have it here: https://docs.openshift.com/serverless/1.30/observability/admin-metrics/serverless-admin-metrics.html?

**Comment by Srivaralakshmi** [comment]
Done incorporating your suggestion, @Preeticp. Thanks for the approval.

**Comment by Srivaralakshmi** [comment]
Hi, @harrietgrace Can you please approve this PR so that I can go ahead and request a merge for the 1.8 branch? TY!

**Comment by harrietgrace** [comment]
Looks good! I don't have permissions on this repo for an actual GitHub approval, so consider this a 👍🏻 from me 😊

**Pull request merged**